### PR TITLE
Update deprecation notice for apps.

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by oliverdunk** [comment]
Is there an issue with some context here, or are these just changes you thought would be nice?

**Comment by jpmedley** [comment]
The old version doesn't provide enough useful information.

File: site/_includes/layouts/namespace-reference.njk [hunk 1]
```diff
@@ -51,7 +51,8 @@
 
         {% if f.platformAppsOnly %}
           <div class="aside aside--warning">
-            This API is part of the deprecated Chrome Apps platform.
+            This API is part of the Chrome Apps platform, which was deprecated in 2020.
+            It is supported only for ChromeOS until Jan 2025.
```
Comment (sebastianbenz) [inline site/_includes/layouts/namespace-reference.njk:55]:
```suggestion
            It is only supported only for ChromeOS enterprise and education customers until Jan 2025.
```

Should we clarify this? Took the info from: https://blog.chromium.org/2021/10/extending-chrome-app-support-on-chrome.html

Comment (oliverdunk) [reply site/_includes/layouts/namespace-reference.njk]: Thanks for the callout Sebastian. @jpmedley, not sure if you were already planning this but we might want to tweak the wording? I don't think it reads great now.

Comment (jpmedley) [reply site/_includes/layouts/namespace-reference.njk]: What do you want it to say?

Comment (sebastianbenz) [reply site/_includes/layouts/namespace-reference.njk]: @oliverdunk have you got any suggestions? 

Update lgtm from my side with the explicit mention of education and enterprise.

Comment (oliverdunk) [reply site/_includes/layouts/namespace-reference.njk]: How about:

>  This API is part of the Chrome Apps platform, which was deprecated in 2020. It remains supported for Enterprise and Education customers on ChromeOS until at least Jan 2025.

The "at least" is based on https://blog.chromium.org/2021/10/extending-chrome-app-support-on-chrome.html unless there was a more recent update I've missed.

File: site/_includes/layouts/reference-landing.njk [hunk 2]
```diff
@@ -32,11 +32,11 @@
...
   <h2 id="platform_apps_apis">Platform Apps APIs</h2>
   <p>
-    These APIs support Chrome Apps on all platforms.
+    These APIs support Chrome Apps.
   </p>
   <div class="aside aside--warning">
-    These APIs are deprecated.
-    Chrome will be removing support for Chrome Apps on all platforms.
+    These APIs are deprecated. They were deprecated in 2020. They are
+    supported only for ChromeOS until Jan 2025.
```
Comment (oliverdunk) [inline site/_includes/layouts/reference-landing.njk:39]:
```suggestion
    supported only for ChromeOS until at least Jan 2025.
```

File: site/_includes/layouts/namespace-reference.njk (hunk 1 above)
Comment (jpmedley) [reply site/_includes/layouts/namespace-reference.njk]: Done.

**Comment by jpmedley** [comment]
@oliverdunk @sebastianbenz I seem to have forgotten about this. PTAL

**Review by sebastianbenz — APPROVED** [review]

File: site/_includes/layouts/reference-landing.njk [hunk 3]
```diff
@@ -32,11 +32,11 @@
 
   <h2 id="platform_apps_apis">Platform Apps APIs</h2>
   <p>
-    These APIs support Chrome Apps on all platforms.
+    These APIs support Chrome Apps.
   </p>
   <div class="aside aside--warning">
-    These APIs are deprecated.
-    Chrome will be removing support for Chrome Apps on all platforms.
+    These APIs are deprecated. They were deprecated in 2020. They are
```
Comment (AmySteam) [inline site/_includes/layouts/reference-landing.njk:38]:
```suggestion
    These APIs were deprecated in 2020. They are
```
More concise

**Review by AmySteam — APPROVED** [review]
Made one suggestion for brevity. Otherwise LGTM 👍

File: site/_includes/layouts/reference-landing.njk [hunk 4]
```diff
@@ -32,11 +32,11 @@
...
   <h2 id="platform_apps_apis">Platform Apps APIs</h2>
   <p>
-    These APIs support Chrome Apps on all platforms.
+    These APIs support Chrome Apps.
   </p>
   <div class="aside aside--warning">
-    These APIs are deprecated.
-    Chrome will be removing support for Chrome Apps on all platforms.
+    These APIs were deprecated in 2020. They are
+    supported only for ChromeOS until at least Jan 2025.
```
Comment (AmySteam) [inline site/_includes/layouts/reference-landing.njk:39]: Sorry missed this, but should we include 
`It remains supported for Enterprise and Education customers on ChromeOS until at least Jan 2025.` in this part also?

Comment (jpmedley) [inline site/_includes/layouts/reference-landing.njk:39]:
```suggestion
    supported only for ChromeOS until at least Jan 2025.
    It remains supported for Enterprise and Education customers on ChromeOS until at least Jan 2025.
```

Comment (jpmedley) [reply site/_includes/layouts/reference-landing.njk:39]: Done.

**Review by AmySteam — APPROVED** [review]

**Comment by jpmedley** [comment]
@matthiasrohmer There are some Percy changes that need to be approved before we can merge this. Are you able to help with that?

**Pull request merged**